source/partials/release_notes/_release-17-9-0.md.erb

```diff
@@ -0,0 +1,74 @@
+<h4>Improvements</h4>
+* <%= link_to_commit '928c5856c2bc73aa92c1a8eb1a86d8094a9fe806', 'Grouping elastic profile by plugin-id', 3772%>
```

Comment:
Grouping elastic agent profiles by type 
We've created a more organized view of elastic agent profiles. You can now see them grouped by the type of elastic agent plugin (Docker Swarm, OpenStack, ECS etc.), making it easier to find a particular profile.